app/views/friendships/_friend.html.erb

```diff
@@ -1,21 +1,16 @@
 <table>
   <tr>
     <th>Name</th>
-    <th><hr/></th>
     <th>Email</th>
-    <th><hr/></th>
     <th>id</th>
   </tr>
 
   <tr>
     <td><%= friend.name%></td>
-    <td><hr/></td>
     <td><%= friend.email%></td>
-    <td><hr/></td>
     <td><%= friend.id%></td>
-    <td><hr/></td>
-    <td><%= link_to 'Unfriend', unfriend_friendship_path(friend), method: :unfriend, data: { confirm: 'Are you sure?' }  if is_friend == '1' %></td>
-    <td><%= link_to 'Cancel Request', cancel_request_friendship_path(friend), method: :cancel_request, data: { confirm: 'Are you sure?' }  if is_friend == '0' %></td>
+    <td><%= link_to 'Unfriend', unfriend_friendship_path(friend), method: :post, data: { confirm: 'Are you sure?' }  if is_friend == true %></td>

```

Comment:
you don't have to do `is_friend == true` for a boolean variable. Rails is intelligent enough to detect that.
so you can replace it with just `if is_friend`

Please change it in other places too